pkg/minishift/registration/redhat.go

```diff
@@ -67,7 +74,16 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 			subscriptionCommand := fmt.Sprintf("sudo -E subscription-manager register --auto-attach "+
 				"--username %s "+
 				"--password '%s' ", param.Username, minishiftStrings.EscapeSingleQuote(param.Password))
+			spinnerView.Start()
+			startTime := time.Now()
 			_, err = registrator.SSHCommand(subscriptionCommand)
+			spinnerView.Stop()
+			if err == nil {
+				fmt.Print("Registration successful ")
+			} else {
+				fmt.Print("Registration unsuccessful ")
+			}
+			util.TimeTrack(startTime, os.Stdout)
```

Comment:
The idea is to use _TimeTrack_ via a defer. 

    defer util.TimeTrack(time.Now(), os.Stdout)